README.md

```diff
@@ -1,50 +1,105 @@
+## .NET 6.0
 
-* [`release/5.0`](https://github.com/dotnet/source-build/tree/release/5.0)
-* [`release/3.1`](https://github.com/dotnet/source-build/tree/release/3.1)
-* [`release/2.1`](https://github.com/dotnet/source-build/tree/release/2.1)
+### Prerequisites
 
-The current branch is a work in progress that produces several .NET SDK components for [Arcade-powered source-build](Documentation/planning/arcade-powered-source-build). Once Arcade-powered source-build is complete, it will allow building a .NET SDK from source directly from the [dotnet/installer](https://github.com/dotnet/installer) repository.
+The dependencies for building .NET 6.0 from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md).
 
-## Using this repository
+### Building
 
-The scripts are written for Bash and supported on macOS and Linux.
+.NET 6.0 is built from source using the [dotnet/installer](https://github.com/dotnet/installer) repo.
+The steps to build vary slightly depending on your distro.  The following set of instructions walk through how to build on Fedora 33.  
 
-> The source-build repository doesn't currently support Windows. See [source-build#1190](https://github.com/dotnet/source-build/issues/1190).
+1. Clone the repo and check out the tag for the desired release.
+    ```bash
+    git clone https://github.com/dotnet/installer
+    cd installer/
+    git checkout v6.0.100-rtm.21527.11
+    ```
+
+3. Create a .NET source tarball.
+
+   ```bash
+   ./build.sh /p:ArcadeBuildTarball=true /p:TarballDir=/path/to/place/complete/dotnet/sources
+   ```
+
+   This fetches the complete .NET source code and creates a tarball at `artifacts/packages/<Release|Debug>/Shipping/`.
+   The extracted source code is also placed at `/path/to/place/complete/dotnet/sources`.
+   The source directory should be outside (and not somewhere under) the installer directory.
+
+3. Prep the source to build on your distro.
+
+    ```bash
+    cd /path/to/complete/dotnet/sources
+    ./prep.sh
+    ```
+
+    This downloads a .NET SDK and a number of .NET packages needed to build .NET from source.
+
+    > On Linux distros other than Fedora 33, an additional bootstrapping step is required.  After running `prep.sh` above, run the following:
+    >
+    > ```bash
+    > tar xf ../packages/archive/Private.SourceBuilt.Artifacts.*.tar.gz
+    > sed -i -E 's|<MicrosoftNETHostModelPackageVersion>6.0.0-rtm.21521.1</|<MicrosoftNETHostModelPackageVersion>6.0.0-rtm.21521.4</|' PackageVersions.props
+    > sed -i -E 's|<MicrosoftNETHostModelVersion>6.0.0-rtm.21521.1</|<MicrosoftNETHostModelVersion>6.0.0-rtm.21521.4</|' PackageVersions.props
+    > tar czf ../packages/archive/Private.SourceBuilt.Artifacts.*.tar.gz *
```

Comment:
```suggestion
    > mkdir ./privateSourceBuiltArtifacts
    > cd ./privateSourceBuiltArtifacts
    > tar xf ../packages/archive/Private.SourceBuilt.Artifacts.*.tar.gz
    > sed -i -E 's|<MicrosoftNETHostModelPackageVersion>6.0.0-rtm.21521.1</|<MicrosoftNETHostModelPackageVersion>6.0.0-rtm.21521.4</|' PackageVersions.props
    > sed -i -E 's|<MicrosoftNETHostModelVersion>6.0.0-rtm.21521.1</|<MicrosoftNETHostModelVersion>6.0.0-rtm.21521.4</|' PackageVersions.props
    > tar czf ../packages/archive/Private.SourceBuilt.Artifacts.*.tar.gz *
    > cd ..
    > rm -r ./privateSourceBuiltArtifacts
```

Reply:
The commands worked fine, but assumed you were in an empty directory that is a peer to `./packages`.  I added the directory commands before and after.